pyproject.toml

```
@@ -82,7 +82,11 @@ pandas = [
 ipywidgets = ["ipywidgets >= 7.7.1", "ipykernel >= 6.2.0"]
 geopandas = ["geopandas >= 0.9.0, < 2.0.0", "Shapely >= 1.8.4, < 3.0.0"]
 ipython = ["ipython >= 7.23.1", "bigquery-magics >= 0.6.0"]
-tqdm = ["tqdm >= 4.7.4, < 5.0.0"]
+matplotlib = [
+  "matplotlib == 3.9.2; python_version == '3.9'",
```

This is quite a restrictive pin. As an "extra" it wouldn't be enforced all the time, but I could see it causing trouble for folks that use the "all" extra. Can we pick a lower bound that's about 2 years old to not force 3.9 users on the latest suppoorted version?

```suggestion
  "matplotlib >= 3.7.1, <= 3.9.2; python_version == '3.9'",
```